vendor/k8s.io/kubernetes/pkg/genericapiserver/storage_factory.go

```diff
@@ -138,6 +138,25 @@ func (s *DefaultStorageFactory) AddCohabitatingResources(groupResources ...unver
 	}
 }
 
+func (s *DefaultStorageFactory) ForceStorageResource(groupResource, toForce unversioned.GroupResource) {
```

Comment:
@deads2k ptal, is this what are you thinking of? I cannot have a groupVersion here to force so I am forcing the groupResource (and the version should be resolved in s.New#StorageForVersion)